src/core/load-fragment.c

```diff
@@ -2462,34 +2463,6 @@ int config_parse_log_extra_fields(
         }
 }
 
-int config_parse_ip_tos(const char *unit,
-                        const char *filename,
-                        unsigned line,
-                        const char *section,
-                        unsigned section_line,
-                        const char *lvalue,
-                        int ltype,
-                        const char *rvalue,
-                        void *data,
-                        void *userdata) {
-
-        int *ip_tos = data, x;
-
-        assert(filename);
-        assert(lvalue);
-        assert(rvalue);
-        assert(data);
-
-        x = ip_tos_from_string(rvalue);
```

Comment:
this one doesn't return negative errno on error, but -1

Reply:
i figure this can just use regular DEFINE_CONFIG_PARSE_ENUM() actually, since the value is pretty much an enum in most of its behaviour, including the "-1 means invalid"  thing